crates/gpui/src/platform/windows/events.rs

```diff
@@ -223,6 +223,30 @@ fn handle_paint_msg(handle: HWND, state_ptr: Rc<WindowsWindowStatePtr>) -> Optio
         state_ptr.state.borrow_mut().callbacks.request_frame = Some(request_frame);
     }
     unsafe { ValidateRect(Some(handle), None).ok().log_err() };
+    const BLEND_PER_PIXEL: BLENDFUNCTION = BLENDFUNCTION {
+        BlendOp: AC_SRC_OVER as u8,
+        BlendFlags: 0,
+        SourceConstantAlpha: 255,
+        AlphaFormat: AC_SRC_ALPHA as u8,
+    };
+    const RGB_BLACK: windows::Win32::Foundation::COLORREF = COLORREF(0); // see https://learn.microsoft.com/en-us/windows/win32/gdi/colorref
+    unsafe {
+        // TODO: pass device contexts so that we have an accurate palette
+        // reference, though this requires plumbing additional events to handle
+        // changes to display properties and so on.
+        UpdateLayeredWindow(
```

Comment:
gah, rerere bit me, this can be removed, will update as soon as i've re-run a test.